Drv/Ip/test/ut/TestTcp.cpp

```diff
@@ -17,8 +16,7 @@ void test_with_loop(U32 iterations) {
     Drv::SocketIpStatus status1 = Drv::SOCK_SUCCESS;
     Drv::SocketIpStatus status2 = Drv::SOCK_SUCCESS;
 
-    U16 port =  Drv::Test::get_free_port();
-    ASSERT_NE(0, port);
+    U16 port =  0; // Choose a port
```

Comment:
Just a small typo with the double space (it's at several places)

Reply:
Will fix!